web-leaflet/browserify/README.md

```diff
@@ -0,0 +1,116 @@
+# Using Browserify with Leaflet (and Esri Leaflet)
+
+> if you don't like to read, just check out the [live sample](http://esri.github.io/developer-support/web-leaflet/browserify/index.html).
+
+## Prerequisites
+
+install [Node.js](https://nodejs.org/en/)/npm.
+
+## Development Steps
+
+1. fork/clone this github repository
+2. navigate into this folder using a command prompt and run `npm install`
+> this will lay down the project [dependencies](https://github.com/Esri/developer-support/blob/master/web-leaflet/browserify/package.json#L8-L18)
```

Comment:
links like this will resolve after the sample is merged.  for now:
s/Esri/jgravois
s/master/leaflet-browserify